rfcs/react-components/convergence/custom-styling.md

```diff
@@ -0,0 +1,390 @@
+# RFC: Custom Styling
+
+## Contributors
+
+Geoff Cox
+Mason Tejera - MADS use cases
+Ben Howell - easier recomposition
+Micah Godbolt - component CSS vars
+
+## Summary
+
+Fluent UI React v9 is built today to adopt the Fluent 2 design as its default style.
+
+Partners migrating from v0/v8 to v9 have brought up that v9's custom styling does not
+provide the same granularity of customization as previous versions.
+Partners are concerned their customers won't be able to deviate from the Fluent 2 design.
```

Comment:
We should rephrase this to something like `customers won't be able to adapt Fluent 2 to their specific scenarios`. Deviate has very negative connotation here and most partners aren't interested in simply doing things differently for no reason. It's just that Microsoft has such a huge range of products and customers it's impossible to address every possible need with a single design. We need to be able to adapt while still having something that feels like a family of products. Most want 90% of what is there and to be able to add small adjustments where it makes sense. The choices we're asking them to make currently in v9 is take this or nothing. Many, unfortunately, will choose nothing or to stay on v8 if they have to compromise their product.